docs/api/router-dashboard.md

```diff
@@ -549,17 +549,22 @@ Calling HTTP `GET` request on this endpoint returns a list of debts.
 `curl 127.0..1:<rita_dashboard_port>/debts`
 
 Format:
-```
+
+```json
 [
- [[mesh_ip,
-   eth_address,
-   wg_public_key],
-  [total_payment_received,
-   total_payment_sent,
-   debt,
-   incoming_payments,
-   debt_buffer]],
-  ...
+  [
+    {
+      "mesh_ip": "a:b:c:d:e:f:g:h",
+      "eth_address": "0x0101010101010101010101010101010101010101",
+      "wg_public_key": "pubkey"
+    },
+    {
+      "total_payment_recieved": "0x0",
```

Comment:
should be received not recieved

Reply:
Thanks for pointing out @asoltys. I'll run `sed` to replace all occurrences globally in repo.